package.json

```diff
@@ -18,6 +18,7 @@
     "@angular/platform-browser": "~12.0.1",
     "@angular/platform-browser-dynamic": "~12.0.1",
     "@angular/router": "~12.0.1",
+    "ngrx-store-localstorage": "^12.0.0",
```

Comment:
what is this used for?

Reply:
I Was watching a tutorial for storing data in localstorage using ngrx so I installed this dependency but not needed. I'll remove it